examples/react-client/videoroom/src/App.tsx

```diff
@@ -67,7 +67,9 @@ function App() {
                 )}
 
                 <div className="absolute bottom-2 left-0 w-full grid place-content-center text-center text-xs z-30">
-                  <p className="bg-slate-100/60 px-1 rounded-sm">{id}</p>
+                  <p className="bg-slate-100/60 px-1 rounded-sm">
+                    {(metadata as { name?: string })?.name ?? id}
```

Comment:
main change

Reply:
that inline type is not the prettiest, but will do the job until we have the predefined metadata typed by the sdk

Reply:
That is my assumption - soon we should have these values typed better, so then it will be fixed.